src/databricks/labs/ucx/queries/progress/main/01_0_migration_readiness.md

```diff
@@ -0,0 +1,5 @@
+# 1. Migration Readiness
```

Comment:
```suggestion
# 1. Migration Progress
```
shouldn't we re-word this with "Progress" instead? "100% progress" here meaning "no more work necessary"